src/comfyui-refiners/README.md

```diff
@@ -18,17 +18,31 @@
 
 ## Installation
 
+1. Navigate to the root of your ComfyUI workspace.
+2. Activate your python virtual environment.
+3. Install the nodes using one of the following methods.
+
+### Comfy Registry (recommended)
+
 The nodes are published at https://registry.comfy.org/publishers/finegrain/nodes/comfyui-refiners.
 
-To easily install the nodes, run the following command:
+To automagically install the nodes, run the following command:
 ```bash
 comfy node registry-install comfyui-refiners
```

Comment:
Last thing: let's add a callout like this here (would not hurt IMO):

![Screenshot 2024-09-19 at 14 07 19](https://github.com/user-attachments/assets/f8804c32-cf6e-49f6-b0e3-84dc06e3d04b)